uint32 servoId
uint32 position
uint32 time